iroh-net/src/relay/map.rs

```diff
@@ -133,6 +145,17 @@ pub struct RelayNode {
     pub stun_port: u16,
 }
 
+impl RelayNode {
+    /// Creates a new relay node from itself.
+    pub fn copied(&self) -> Self {
```

Comment:
I don't know what's the expected naming or way to do this properly. I just needed to get it out of the arc and this was cheap and easy enough.

Reply:
no need for this method, just use this to get it out of the arc: https://doc.rust-lang.org/std/sync/struct.Arc.html#method.unwrap_or_clone

Reply:
Figured there was a saner way to do this.